modules/gdpr_fields/gdpr_fields.links.task.yml

```diff
@@ -0,0 +1,6 @@
+gdpr_fields.fields_list:
+  route_name: gdpr_fields.fields_list
+  title: 'GDPR field list'
+  base_route: entity.field_storage_config.collection
+  base_route: gdpr_tasks.summary
```

Comment:
The gdpr_tasks.summary route does not exist (in the context of the 8.x-1.x branch and this PR).

Reply:
Yes I thought it would be helpful to have an intro page maybe with some help text and useful links or settings for the module, but so far I have not thought of what to put here. I am happy to remove this for now, but then we will need a new default tab under the GDPR toolbar item. Since the GDPR module doesn't provide any pages I am not sure what to make the default tab. Maybe the field list?